docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx

```diff
@@ -23,8 +23,8 @@ To test API in your local instance, you have the following pre-requisites:
         Start the cal.com server using `yarn dev` on localhost and create the test API keys by visiting `/settings/developer/api-keys`
     </Step>
     <Step title="Start API server and test API">
-        Start the API server by running `yarn workspace @calcom/api dev` and start testing your API locally
+        Start the API server by running `yarn workspace @calcom/api dev` (or `yarn workspace @calcom/api-v2 dev` if api-v2) and start testing your API locally
```

Comment:
```suggestion
        Start the API server by running `yarn workspace @calcom/api dev` and start testing your API locally
```

let's keep it as it is for now.

Reply:
@sathuli2710 please sign the CLA as well.

Reply:
Sorry I am new to open source, can you please guide me to sign CLA. Looking forward to contribute more in the future.

Reply:
nvm it's merged now.